cli/ballerina-tool/src/main/java/org/ballerinalang/tool/util/ToolUtil.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This is an already existing file. So we shouldn't change the year if I'm not mistaken.

Reply:
File was added last month and 2017 was not correct.